pkgs/development/python-modules/treescope/default.nix

```diff
@@ -0,0 +1,65 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+
+  # build-system
+  flit-core,
+
+  # dependencies
+  numpy,
+
+  # optional-dependencies
+  ipython,
+  jax,
+  palettable,
+
+  # tests
+  absl-py,
+  jaxlib,
+  pytestCheckHook,
+  torch,
+}:
+
+buildPythonPackage rec {
+  pname = "treescope";
+  version = "0.1.5";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "google-deepmind";
+    repo = "treescope";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-+Hm60O9tEXIiE0av1O0BsOdMln4e1s7ijb3WNiQ74jE=";
+  };
+
+  build-system = [ flit-core ];
+
+  dependencies = [ numpy ];
+
+  optional-dependencies = {
+    notebook = [
+      ipython
+      jax
+      palettable
+    ];
+  };
+
+  pythonImportsCheck = [ "treescope" ];
+
+  nativeCheckInputs = [
+    absl-py
+    jax
+    jaxlib
+    pytestCheckHook
+    torch
+  ] ++ optional-dependencies.test;
```

Comment:
eval failure is now due to this attr missing

Reply:
Fixed.